### DISCUSSION TITLE
RuntimeError: status command squeue fails to execute

### DISCUSSION THREAD

**Discussion by goodluck-xyz** [deepmodeling/dpgen#1042] [Q&A]
When I run dpgen in new machine, I meet the error `RuntimeError: status command squeue fails to execute`
the error log as fllow
[slurm-115.txt](https://github.com/deepmodeling/dpgen/files/10067091/slurm-115.txt)
 the machine.json as fllow
[40jifang_machine.txt](https://github.com/deepmodeling/dpgen/files/10067076/40jifang_machine.txt)

**Comment by HuangJiameng** [comment]
Maybe you can refer to https://stackoverflow.com/questions/55290596/solving-slurm-sbatch-error-batch-job-submission-failed-requested-node-config, can try reducing `cpu_per_node` in `machine.json`

**Reply by goodluck-xyz** [reply]
Thank you
